gemma-core/src/main/java/ubic/gemma/core/expression/experiment/ExpressionExperimentMetaFileType.java

```diff
@@ -0,0 +1,69 @@
+package ubic.gemma.core.expression.experiment;
+
+import ubic.gemma.model.expression.experiment.ExpressionExperiment;
+
+import java.io.File;
+
+/**
+ * Types of metadata files that can be attached to an {@link ExpressionExperiment}.
+ */
+public enum ExpressionExperimentMetaFileType {
```

Comment:
The big story here is that I converted the MetaFileType class into an enumeration.